bug-1674187-pref-holdback-study-for-quan.toml

```diff
@@ -0,0 +1,59 @@
+[metrics]
+daily = ["active_hours", "searchmode_counts", "searchalias_counts", "searchmode_amazon_counts", "searchalias_amazon_counts", "searchoneoff_amazon_counts"]
+weekly = ["ad_clicks", "search_count", "organic_search_count", "tagged_search_count", "tagged_follow_on_search_count"]
+overall = ["active_hours", "uri_count", "ad_clicks", "search_count"]
+
+[metrics.unenroll.statistics.binomial]
+
+[metrics.active_hours.statistics.bootstrap_mean]
+[metrics.active_hours.statistics.deciles]
+
+[metrics.uri_count.statistics.bootstrap_mean]
+[metrics.uri_count.statistics.deciles]
+
+[metrics.ad_clicks.statistics.bootstrap_mean]
+[metrics.ad_clicks.statistics.deciles]
+
+[metrics.search_count.statistics.bootstrap_mean]
+[metrics.search_count.statistics.deciles]
+
+[metrics.retained]
+select_expression = "COALESCE(SUM(pings_aggregated_by_this_row), 0) > 0"
+data_source = "clients_daily"
+
+[metrics.retained.statistics.binomial]
```

Comment:
These declarations are also redundant with the default configuration, so you can omit them